corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -344,6 +346,8 @@ def _v2_fixtures(self, restore_user, report_configs):
     def report_config_to_fixture(self, report_config, restore_user):
         def _row_to_row_elem(deferred_fields, filter_options_by_field, row, index, is_total_row=False):
             row_elem = E.row(index=str(index), is_total_row=str(is_total_row))
+            if toggles.ADD_ROW_INDEX_TO_MOBILE_UCRS.enabled(restore_user.domain):
+                row_elem.append(E('row_index', str(index)))
```

Comment:
nit: it looks like this `_row_to_row_elem` is identical for both v1 and v2, so you could probably pull it out.

Reply:
v1 is using E but v2 is using E.column for columns which makes them slightly different. The signature to both these methods is different as well, hence just passing them as an argument to us to a a function won't do.